src/core_plugins/kibana/public/visualize/listing/visualize_landing_page_table.js

```diff
@@ -170,18 +212,26 @@ export class VisualizeLandingPageTable extends React.Component {
       <LandingPageToolBar
         filter={filter}
         onFilter={this.onFilter}
-        pagerState={pager}
+        startNumber={0}
+        endNumber={0}
+        totalItems={10}
+        hasPreviousPage={this.hasPreviousPage}
+        hasNextPage={this.hasNextPage}
         onNextPage={this.turnToNextPage}
         onPreviousPage={this.turnToPreviousPage}
         actionButtons={this.getActionButtons()}/>
       {
         this.getTableContents()
       }
       <LandingPageToolBarFooter
-        pagerState={pager}
+        startNumber={0}
+        endNumber={0}
+        totalItems={10}
+        hasPreviousPage={this.hasPreviousPage}
+        hasNextPage={this.hasNextPage}
```

Comment:
Yes, maybe the `LandingPageToolBar` encapsulation is unnecessary. Just composing `KuiSectionedToolBar`, `KuiToolBarSearchBox` and `KuiToolBarPager` might be enough. Roughly:

```javascript
  render() {
    const { filter, pager, selectedIds } = this.state;
    const pager = (
      <KuiToolBarPager
        startNumber={0}
        endNumber={0}
        totalItems={10}
        hasPreviousPage={this.hasPreviousPage}
        hasNextPage={this.hasNextPage}
      />
    );

    return (
      <div>
        <KuiSectionedToolBar>
          <KuiToolBarSearchBox filter={filter} onFilter={onFilter}/>
          { actionButtons }
          { pager }
        </KuiSectionedToolBar>
        { this.getTableContents() }
        <div className="kuiToolBarFooter">
          <KuiSelectedItemsFooterSection selectedItemsCount={selectedIds.length} />
          { pager }
        </div>
      </div>
    );
  }
```